<launch>
   <arg name="base_link" default="base_link" />
   <arg name="laser_scan" default="front_laser_frame" />
   <node pkg="sicknav350" type="sicknav350_node" name="sicknav350_node" output="screen" >
     <param name="publish_tf" value="false" />
     <param name="publish_odom_" value="false" /> 
     <param name="publish_scan" value="true" />
     <!--added by chq -->
    <param name="ipaddress" value="172.16.41.121" />
    <param name="port" value="2111" />
    <param name="fixed_frame_id" value="$(arg base_link)" />
    <param name="frame_id" value="$(arg laser_scan)" />
   </node>
   <node pkg="tf" type="static_transform_publisher" name="static_transform_publisher" args="0.1 0 0.75 0 0 0 $(arg base_link) $(arg laser_scan) 100" />
   <!--node pkg="rviz" type="rviz" name="rviz" args="-d $(find sicknav350)/launch/nav350_intensity.rviz" /-->
</launch>
